CIP-1776/README.md

```diff
@@ -0,0 +1,84 @@
+
+This proposal proceeds on the axiom that the flaws in CIP 1694 are self-evident (which indeed they are) and proposes a substantially more viable alternative.
+
+The core issues summarized:
+
+1. Arbritrary complexity: The proposal introduces several new concepts and mechanisms, such as the Constitutional Committee, DReps, and new types of voting certificates, which makes the system unneccisarily complex for stakeholders - this will reduce overall participation in governance.
+
+2. Poor intentive structure: The proposal mentions that DReps need to be compensated for their work, but it is not clear how this will be done or what the exact mechanisms will be. It fails to leverage the existing incentive structures of the Cardano blockchain. 
+
+3. Bootstrapping problem: The proposal relies on the presence of sufficient DReps to transition the system from its current state to decentralized governance. It is not clear how this will be achieved in practice.
+
+4. Perverse incentives: The proposal as put forward by IOG entrenches existing powers rather than truly grant control over the Cardano Blockchain to the community. 
+
+5. Exclusionary Representation: Current discussions fo CIP 1694 effectively exclude members of the developing word from access by introducing a threshold to become a DRep. 
+
+The author proposes a new alternative, a non-incentivized iterative block approval voting system to manage treasury disbursement (that would be a variant of multi-winner approval voting modified to resolve the Burr dilemma) and a three-house structure for overarching governance actions. This alternative would be more decentralized, and the ratification of governance actions would be the responsibility of SPOs, Dreps, and a highly modified constitutional committee herein the ‘SC’ or Supreme Court. Let us apply Occam's Razor, put simply “the simplest solution is almost always the best". A system of arbitrary complexity is often developed those of great intelligence - this issue plagues particle physics where increasingly complex models are proposed as a refutation to the standard model, requiring increasingly complex experimentation which ultimately results in increasingly expensive refutation of the underlying theory. This problem manifests in almost all fields of academia. 
+
+Further, the author proposes an alternative form of Cardano governance, radically improving upon the experimental Project Catalyst which in effect resolves systematic issues uncovered during the priciple experiment while incoperating all treasury actions as a formalized governance action. 
+
+----
+##A series of bullet points / template
+
+- System can be divided simply into three houses
+    - House of Dreps 
+    - Congress of SPOs 
+    - Supreme Court
```

Comment:
I'm really interested, as you flesh out this proposal, to learn about the supreme court.  Does this boil down to simply a name change? in what ways will it materially differ from the constitutional committee?

The argument that simply naming it one thing will lead to "the same central planning committee of the failed communist states" holds basically 0 weight.  The outlined powers for the constitutional committee include none of the central planning powers of those "failed communist states". If you can strengthen that connection, that would be interesting, but as is, it just comes off as a knee-jerk reactionary objection based on irrelevant biases.

Reply:
Thanks @Quantumplation - and I appreciating you going to the effort of seeking out this CIP. As I have mentioned I've got very serious scheduling issues that I have to address this month but I'm confident you'll enjoy reading V1 of CIP 1776 when it is available. Target date March 31/April 1 - I understand the urgency and as soon as I finish moving into my new office and can sort though my written I'll upload the existing V1 draft.